Draft improvements from assistant conversations: run moves and nested config (#2830)
wandb/docs: models/runs/manage-runs.mdx

When you move a run, historical artifacts associated with it are not moved. To move an artifact manually, you can use the [`wandb artifact get`](/models/ref/cli/wandb-artifact/wandb-artifact-get/) SDK command or the [`Api.artifact` API](/models/ref/python/public-api/api/#artifact) to download the artifact, then use [`wandb artifact put`](/models/ref/cli/wandb-artifact/wandb-artifact-put/) or the `Api.artifact` API to upload it to the run's new location.
</Note>

## Delete runs programmatically

> To delete a run from the Python Public API, use [`Run.delete()`](/models/ref/python/public-api/run#method-run-delete). This is the only run-management operation available through the Public API; moving runs between projects must be performed in the W&B App.

Reviewer: ❌ **Refuted** — the source says this is wrong.

```suggestion
To delete a run from the Python Public API, use [`Run.delete()`](/models/ref/python/public-api/run#method-run-delete). The Public API also supports other run-management operations, but moving runs between projects requires the W&B App.
```

**Source says:** `Run.update()`, `Run.save()`, and `Run.update_state()` are also public run-management methods. · **Source:** [wandb:runs.py#L937](https://github.com/wandb/wandb/blob/a2cb871814a97cb7094fb34f014c3cf5d1287c34/wandb/apis/public/runs.py#L937-L1018)

_🤖 CoreWeave Docs Team's Validator — refuted_
<!-- cw-docs-validator -->
